shared/commonUtil.ts

```diff
@@ -39,7 +39,7 @@ export interface Argon2iConfig {
   mode: string;
 }
 
-export default function testIDProps(id) {
+export default function testIDProps(id: string | undefined) {
```

Comment:
testIdProps will never be undefined. It will be string always, so please remove undefined from type